docs/commands.md

```diff
@@ -408,8 +408,11 @@ cpl run -a $APP_NAME --entrypoint /app/alternative-entrypoint.sh -- rails db:mig
 - Creates an app and all its workloads
 - Specify the templates for the app and workloads through `setup_app_templates` in the `.controlplane/controlplane.yml` file
 - This should only be used for temporary apps like review apps, never for persistent apps like production (to update workloads for those, use 'cpl apply-template' instead)
-- Automatically binds the app to the secrets policy, as long as both the identity and the policy exist
-- Use `--skip-secret-access-binding` to prevent the automatic bind
+- Automatically creates a secret and policy for the app
+  (by default, the secret name is "{APP_PREFIX}-secrets" and the policy name is "{APP_PREFIX}-secrets-policy",
+  but they can be customized through `secrets_name` and `secrets_policy_name` in the `.controlplane/controlplane.yml` file)
+- Automatically binds the app to the secrets policy, as long as the identity exists
+- Use `--skip-secret-access-binding` to prevent the automatic creation/binding
```

Comment:
Changes automatically generated based on `setup_app.rb`, so see that file instead.